src/goto-programs/read_bin_goto_object.cpp

```diff
@@ -100,7 +103,10 @@ bool read_bin_goto_object(
     irept t;
     dstring fname = irepconverter.read_string(in);
     gfconverter.convert(in, t);
-    goto_functiont &f = functions.function_map[fname];
+    auto it = functions.function_map.find(fname);
+    if(it == functions.function_map.end())
+      functions.function_map.emplace(fname, message_handler);
+    goto_functiont &f = functions.function_map.at(fname);
```

Comment:
Because I neeeded to pass the msg parameter to the constructor. For what I understood if you try to access an element in a map with the `[]` operator it will create the element if it doesn't exist (which caused compilation issues).